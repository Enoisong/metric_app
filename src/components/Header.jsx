import { Link } from 'react-router-dom';
import back from './image/ba.png';
import microphone from './image/bd.png';
import setting from './image/be.png';

const Header = () => (
  <div className="top-header">
    <Link to="/"><img src={back} alt="Back Icon" className="back-im" /></Link>
    <h1>Population Metrics</h1>
    <div className="right-icon">
      <img src={microphone} alt="Microphone icon" />
      <img src={setting} alt="Setting icon" className="set-img" />
    </div>
  </div>
);

export default Header;
