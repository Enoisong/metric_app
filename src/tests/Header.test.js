import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from '../components/Header';

describe('Header', () => {
  it('Should renders correctly', () => {
    const component = renderer.create(
      <Router>
        <Header />
      </Router>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
