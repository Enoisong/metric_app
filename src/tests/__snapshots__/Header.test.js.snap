// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should renders correctly 1`] = `
<div
  className="top-header"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Back Icon"
      className="back-im"
      src="ba.png"
    />
  </a>
  <h1>
    Population Metrics
  </h1>
  <div
    className="right-icon"
  >
    <img
      alt="Microphone icon"
      src="bd.png"
    />
    <img
      alt="Setting icon"
      className="set-img"
      src="be.png"
    />
  </div>
</div>
`;
